hpsklearn/estimator/estimator.py

```diff
@@ -277,6 +282,7 @@ def fit_iter(self, X, y,
                      EX_list=EX_list,
                      valid_size=valid_size,
                      n_folds=n_folds,
+                     kfolds_group = kfolds_group,
```

Comment:
```suggestion
                     kfolds_group=kfolds_group,
```
My bad. My previous review on this line was incorrect.

Saturday it's on my list to properly look at this pr